performance_regression_test.py

```diff
@@ -99,7 +105,35 @@ def get_test_xml(self, result):
 
         return test_content
 
-    def display_results(self, results):
+    def generate_stats_json(self, results, cmds=[]):
+        metrics = {}
+
+        for p in self.params.iteritems():
+            metrics[p[1]] = p[2]
+
+        # we use cmds
+        del metrics['stress_cmd']
+        for cmd in cmds:
+            metrics = self.add_stress_cmd_params(metrics, cmd)
+            metrics = self.add_stress_cmd_params(metrics, cmd)
+
+        metrics['test_name'] = self.params.id.name
+        metrics['stats'] = results
```

Comment:
guys, the code generate 2 json with stats:
https://gist.github.com/abvgedeika/8a1c57f21ade86982809c66051549c59  stats by nodes, we don't put this version into ES(only for verification and visibility now)
https://gist.github.com/abvgedeika/542802d74809ae9203a3a6944d64a300  - stats that we send in ES woth the avg of the aggregated result